### ISSUE TITLE
"Get directions" links with the same name 

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#24751]
# Issue Description
This issue was found during testing in preparation for staging review. 

**Defect**
Links with the same name have a similar purpose. Locations on this page all have the same link. 
Related: when navigating by keyboard, the name of the location does not receive focus. We may be able to improve both issues simultaneously

![get directions link defect](https://user-images.githubusercontent.com/55411834/118510262-065a8780-b6ee-11eb-8b46-9382e76f77fa.png)

**Code**

`<a href="https://maps.google.com?saddr=Current+Location&amp;daddr=3500 Ludington Street, Escanaba, MI 49829" target="_blank" rel="noopener noreferrer"> Get directions </a>`

---
## Tasks
- [ ] Add name to link announcement (Get directions to [Vet Center name])

## Acceptance Criteria
- [ ] Veterans using screenreaders have context when navigating by keyboard and can select the link associated with the intended facility

**Comment by mmiddaugh** [comment]
Closing this issueas it is a duplicate to the more thoroughly documented #25006
